docs/developer-docs/latest/developer-resources/plugin-api-reference/admin-panel.md

```diff
@@ -411,6 +411,16 @@ A plugin has 2 different ways of injecting a component:
 * to inject a component from a plugin into another plugin's injection zones, use the `injectComponent()` function
 * to specifically inject a component into one of the Content Manager's [predefined injection zones](#using-predefined-injection-zones), use the `injectContentManagerComponent()` function instead
 
+Both the `injectComponent()` and `injectContentManagerComponent()` methods accept the following arguments:
```

Comment:
👍 although I would specify that for the injectContentManagerComponent it can only be used to inject components in the content manager's view but I guess the naming is enough.